Makefile

```diff
@@ -70,3 +71,12 @@ beats-dashboards:
 .PHONY: docs
 docs:
 	sh libbeat/scripts/build_docs.sh ${PROJECTS}
+
+.PHONY: package
+package:
+	$(MAKE) -C libbeat package-setup
+	$(foreach var,$(BEATS),SNAPSTHOT=$SNAPSHOT $(MAKE) -C $(var) package || exit 1;)
```

Comment:
Btw, parallelism should be only an option, I don't think it will be used in the unified build.